state/validators.md

```diff
@@ -0,0 +1,38 @@
+# Validator Selection Scheme
+
+## Abstract
+
+This document elucidates the validator selection mechanism as outlined in the [Rollkit repository](https://github.com/rollkit/rollkit). The scheme establishes an initial set of validators at the genesis of the blockchain, then leverages a priority-based system to determine the proposer of new blocks, ensuring a decentralized and fair block proposal mechanism.
+
+## Protocol/Component Description
+
+The protocol initializes with a predefined set of validators as stated in the `GenesisDoc`. These validators are part of the `GenesisValidator` struct which includes their address, public key, voting power, and name. Post genesis, the validator set might change, and is managed through Tendermint's `ValidatorSet` and `State` structures.
```

Comment:
This description should mention that size of the set of validators is 1.

Instead of `Tendermint` here, `CometBFT` should be used since Rollkit imports `CometBFT` currently, not `Tendermint`.

Reply:
Updated.